cylc/flow/scripts/set.py

```diff
@@ -18,41 +18,59 @@
 
 """cylc set [OPTIONS] ARGS
 
-Command to manually set task prerequisites and outputs in running workflows.
-
-By default, it sets all required outputs (including "submitted", "started" and
-"succeeded" even if they are optional).
-
-Setting task prerequisites:
-  - contributes to the task's readiness to run, and
-  - promotes it to the scheduler's active task pool
-
-Note --pre=all also promotes parentless tasks (with no task-prerequisites) to
-the active pool where clock and xtriggers become active. This is needed to
-start a new flow that continues to future cycle points, if you need the first
-parentless tasks in the new flow to wait on clock or xtriggers before running.
-
-Setting task outputs:
-  - contributes to a task's completion, and
-  - spawns downstream tasks that depend on those outputs
-
-Note setting final outputs ("succeeded", "failed", "expired") also sets task
-state. Setting the "started" and "submitted" outputs spawns downstream tasks
-that depend on them but does not affect task state, because there is no
-running job.
-
-Implied outputs are set automatically:
-  - "started" implies "submitted"
-  - "succeeded" and "failed" imply "started"
-  - custom outputs and "expired" do not imply other outputs
-
-For custom outputs, use the output names not the associated task messages:
-[runtime]
-  [[my-task]]
-    # ...
-    [[[outputs]]]
-      # <output-name> = <task-message>
-      x = "file x completed and archived"
+Command to manually set task prerequisites and outputs, and xtriggers.
+
+By default, this sets all required outputs plus the "submitted", "started",
```

Comment:
Probably worth being super clear that you can't set outputs for an xtrigger?

```suggestion
By default, this sets all required outputs of a task plus the "submitted", "started",
```

Reply:
Done.